appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - edm install -y numpy cython pytest mock h5py
 
   # Install pysph related dependencies.
-  - edm run -- pip install -r requirements.txt
+  - edm run -- pip install .[test]
```

Comment:
Does this also install the package (i.e. pysph) or does it just install the deps?

Reply:
Right. It installs the package too. The next statement is redundant.